node/impl/src/main/java/eu/cloudnetservice/node/impl/service/defaults/AbstractService.java

```diff
@@ -411,6 +412,19 @@ public void restart() {
     this.updateLifecycle(ServiceLifeCycle.RUNNING);
   }
 
+  @Override
+  public Collection<String> consoleSuggestion(@NonNull String line) {
+    var listPropertyType = TypeFactory.parameterizedClass(List.class, String.class);
```

Comment:
I think this method should return a future instead so that the caller can better handle errors and apply timeouts.